samples/excel/01-basics/multiple-property-set.yaml

```diff
@@ -0,0 +1,155 @@
+order: 3
+id: excel-multiple-property-set
+name: Multiple Property Set
+description: Setting multiple properties at once with the rich API object set() method.
+author: OfficeDev
+host: EXCEL
+api_set:
+    ExcelApi: 1.4
+script:
+    content: |
+        $("#setup").click(setup);
+
+		$("#set-multiple-properties-with-object").click(setMultiplePropertiesWithObject);
+		$("#copy-properties-from-range").click(copyPropertiesFromRange);
+
+		async function setMultiplePropertiesWithObject() {
+			try {
+				await Excel.run(async (context) => {
+					const sheet = context.workbook.worksheets.getItem("Sample");
+
+					const range = sheet.getRange("B2:E2");
+					range.set({
+						format: {
```

Comment:
Well, it currently is designed to change only the header range. Do you want an example that changes the styling and number format of the data range?